main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This loop currently ends early, requirements state user should be able to create 10 stories.
```suggestion
while (loop <= 10):
```